### PR TITLE
Feature/extend history

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**What change(s) does this PR introduce?** 

- introduces a `jira_issue_history_buffer` variable to extend the month-long buffer in the issue_field_history model 
- set issue daily history columns to lowercase to avoid ambiguous column errors in Snowflake
- roll back spark changes
- introduce dbt 20 changes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide explanation how the change is non breaking below.)

**Is this PR in response to a previously created Issue**

- [ ] Yes, Issue [link issue number here]
- [x] No - from customer tho  

**How did you test the PR changes?** 

 
- [x] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🐯 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/jira__issue_enhanced.sql
Comment (fivetran-joemarkiewicz) [inline models/jira__issue_enhanced.sql:13]:
```suggestion
    {{ issue_data_columns_clean.append(k.name)|default("", True)  }}
```

File: packages.yml
Comment (fivetran-jamie) [inline packages.yml:2]: change before merging

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM - Can we be sure to hold off releasing this package until the source is released on the hub. I want to make sure circle passes with the jira_source hub version once it is available. Then we can release this one onto the hub.

**Pull request merged**
